modules/ui/src/Form/StockTransactions1.php

```diff
@@ -43,14 +43,6 @@ public function buildForm(array $form, FormStateInterface $form_state) {
   /**
    * {@inheritdoc}
    */
-  public function validateForm(array &$form, FormStateInterface $form_state) {
```

Comment:
The code sniffer complains about: "Possible useless method overriding detected (Generic.CodeAnalysis.UselessOverridingMethod.Found)" and issues a warning, hence builds failing. If we want to keep some or more of this "useless overrides" to put our own validation logic later in, we can add some annotations, so that the code sniffer do not complain about them or we could add our own code sniffer config file and entirely disable that rule.

But maybe we should remove them for now and open an issue or multiple ones to keep track of the task. I think we will miss things if we have ToDo's in code without corresponding issues on d.o.